hybig/sizes.py

```diff
@@ -422,7 +422,7 @@ def find_closest_resolution(
 
     """
     best_info = None
-    smallest_diff = np.Infinity
+    smallest_diff = np.inf
```

Comment:
This should have been in #38 😬

Reply:
Hmm.  I don't know now.  I thought I had an error because of it. but I can't even find one for lint or ruff now. I think it's deprecated in any case. https://numpy.org/devdocs/numpy_2_0_migration_guide.html